modin/pandas/test/test_io.py

```diff
@@ -248,7 +262,7 @@ def _csv_file_maker(
                 "delimiter": delimiter,
                 "doublequote": doublequote,
                 "escapechar": escapechar,
-                "lineterminator": line_terminator,
+                "lineterminator": line_terminator if line_terminator else os.linesep,
```

Comment:
Why use the ternary operator here?

Reply:
`lineterminator` is common parameter for `pandas.to_csv` and for `csv.reader/writer`. Parameter sets default value for `pandas` case (None), but for `csv` we have to set exact value.